hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -608,7 +608,7 @@
   /**
    * Last row in a table.
    */
-  public static final byte [] EMPTY_END_ROW = EMPTY_START_ROW;
+  public static final byte [] EMPTY_END_ROW = EMPTY_BYTE_ARRAY;
```

Comment:
We already have `EMPTY_START_ROW = EMPTY_BYTE_ARRAY`. By this change, do you mean to improve readability?

Reply:
I think all the changes here are for readability. As even for the above EMPTY_START_ROW to EMPTY_END_ROW change, there is no difference on behavior as they are all EMPTY_BYTE_ARRAY...

Reply:
Yup, agree. We can merge this after PR title change.